client/components/Nav.jsx

```diff
@@ -9,9 +9,9 @@ const Nav = () => {
           <h2>SW</h2>
         </a>
       </div>
-      <div className='navbar-menu'>
+      <div className='navbarmenu'>
         {/* To be replaced by an Auth */}
-        <Link to='/login' className='button'>login</Link>
+        <Link to='/login' className='button' id='login-button'>login</Link>
```

Comment:
Let's use a class here instead of an id unless we really need to.

Reply:
Removed id